TelegramTest.[mp3|mp4|pdf] fail with Camel 4 and Quarkus 3

```
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.apache.camel.quarkus.component.telegram.it.TelegramTest
[INFO] [stdout] 2023-02-07 10:43:52,612 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (pool-2-thread-1) Starting WireMockServer
[INFO] [stdout] 2023-02-07 10:43:52,662 INFO  [org.apa.cam.qua.tes.moc.bac.MockBackendUtils] (pool-2-thread-1) Mock backend will be used
[INFO] [stdout] 2023-02-07 10:43:52,714 INFO  [org.apa.cam.qua.tes.AvailablePortFinder] (pool-2-thread-1) org.apache.camel.quarkus.test.wiremock.WireMockTestResourceLifecycleManager reserved port 46359
[INFO] [stdout] 2023-02-07 10:43:53,328 INFO  [org.ecl.jet.ser.Server] (pool-2-thread-1) jetty-11.0.12; built: 2022-09-14T02:38:00.723Z; git: d5b8c29485f5f56a14be5f20c2ccce81b93c5555; jvm 19.0.2+7
[INFO] [stdout] 2023-02-07 10:43:53,351 INFO  [org.ecl.jet.ser.han.ContextHandler] (pool-2-thread-1) Started o.e.j.s.ServletContextHandler@4e17e61d{/__admin,null,AVAILABLE}
[INFO] [stdout] 2023-02-07 10:43:53,362 INFO  [org.ecl.jet.ser.han.Con.ROOT] (pool-2-thread-1) RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.StubRequestHandler. Normalized mapped under returned 'null'
[INFO] [stdout] 2023-02-07 10:43:53,363 INFO  [org.ecl.jet.ser.han.ContextHandler] (pool-2-thread-1) Started o.e.j.s.ServletContextHandler@411db415{/,null,AVAILABLE}
[INFO] [stdout] 2023-02-07 10:43:53,372 INFO  [org.ecl.jet.ser.AbstractConnector] (pool-2-thread-1) Started NetworkTrafficServerConnector@a2e518c{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:46359}
[INFO] [stdout] 2023-02-07 10:43:53,374 INFO  [org.ecl.jet.ser.Server] (pool-2-thread-1) Started Server@6844e19c{STARTING}[11.0.12,sto=1000] @5361ms
[INFO] [stdout] 2023-02-07 10:43:53,374 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (pool-2-thread-1) WireMock started on http://localhost:46359
[INFO] [stdout] 2023-02-07 10:43:53,444 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.https.test-port" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
[INFO] [stdout] 2023-02-07 10:43:53,887 INFO  [org.apa.cam.qua.cor.CamelBootstrapRecorder] (main) Bootstrap runtime: org.apache.camel.quarkus.main.CamelMainRuntime
[INFO] [stdout] 2023-02-07 10:43:53,888 INFO  [org.apa.cam.mai.MainSupport] (main) Apache Camel (Main) 4.0.0-M1 is starting
[INFO] [stdout] 2023-02-07 10:43:53,940 INFO  [org.apa.cam.mai.BaseMainSupport] (main) Auto-configuration summary
[INFO] [stdout] 2023-02-07 10:43:53,941 INFO  [org.apa.cam.mai.BaseMainSupport] (main)     [MicroProfilePropertiesSource] camel.component.telegram.baseUri=http://localhost:46359
[INFO] [stdout] 2023-02-07 10:43:53,942 INFO  [org.apa.cam.mai.BaseMainSupport] (main)     [MicroProfilePropertiesSource] camel.component.telegram.authorizationToken=xxxxxx
[INFO] [stdout] 2023-02-07 10:43:54,062 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 4.0.0-M1 (camel-1) is starting
[INFO] [stdout] 2023-02-07 10:43:54,146 INFO  [org.apa.cam.com.web.WebhookEndpoint] (main) Registering webhook for endpoint: telegram://bots
[INFO] [stdout] 2023-02-07 10:43:54,341 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Routes startup (started:1)
[INFO] [stdout] 2023-02-07 10:43:54,341 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main)     Started route1 (webhook:telegram:bots)
[INFO] [stdout] 2023-02-07 10:43:54,341 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 4.0.0-M1 (camel-1) started in 348ms (build:0ms init:70ms start:278ms)
[INFO] [stdout] 2023-02-07 10:43:54,424 INFO  [io.quarkus] (main) camel-quarkus-integration-test-telegram 3.0.0-SNAPSHOT on JVM (powered by Quarkus 3.0.0.Alpha3) started in 3.444s. Listening on: http://localhost:39889
[INFO] [stdout] 2023-02-07 10:43:54,424 INFO  [io.quarkus] (main) Profile test activated. 
[INFO] [stdout] 2023-02-07 10:43:54,425 INFO  [io.quarkus] (main) Installed features: [camel-attachments, camel-core, camel-platform-http, camel-rest, camel-telegram, cdi, resteasy, resteasy-jackson, smallrye-context-propagation, vertx]
[INFO] [stdout] 2023-02-07 10:43:54,444 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (main) Injecting WireMockServer for field server
[INFO] [stdout] 2023-02-07 10:43:56,478 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-0) Received telegram messages: test
[INFO] [stdout] 2023-02-07 10:43:56,556 INFO  [org.apa.cam.qua.com.tel.it.TelegramTest] (main) Telegram Bot received messages: test
[INFO] [stdout] 2023-02-07 10:43:56,983 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-0) Received telegram messages: test
[INFO] [stdout] 2023-02-07 10:43:56,986 INFO  [org.apa.cam.qua.com.tel.it.TelegramTest] (main) Telegram Bot received messages: test
[INFO] [stdout] 2023-02-07 10:43:57,488 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-0) Received telegram messages: test
[INFO] [stdout] 2023-02-07 10:43:57,490 INFO  [org.apa.cam.qua.com.tel.it.TelegramTest] (main) Telegram Bot received messages: test
[INFO] [stdout] 2023-02-07 10:43:57,991 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-0) Received telegram messages: test
[INFO] [stdout] 2023-02-07 10:43:57,995 INFO  [org.apa.cam.qua.com.tel.it.TelegramTest] (main) Telegram Bot received messages: test
[INFO] [stdout] 2023-02-07 10:43:58,496 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-0) Received telegram messages: test
[INFO] [stdout] 2023-02-07 10:43:58,499 INFO  [org.apa.cam.qua.com.tel.it.TelegramTest] (main) Telegram Bot received messages: test
[INFO] [stdout] 2023-02-07 10:43:58,507 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (main) Injecting WireMockServer for field server
[INFO] [stdout] 2023-02-07 10:44:28,574 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (main) Injecting WireMockServer for field server
[INFO] [stdout] 2023-02-07 10:44:58,610 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (main) Injecting WireMockServer for field server
[INFO] [stdout] 2023-02-07 10:45:28,633 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (main) Injecting WireMockServer for field server
[INFO] [stdout] 2023-02-07 10:45:28,683 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-3) Sent a message to telegram [B@43c82883
[INFO] [stdout] 2023-02-07 10:45:28,699 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (main) Injecting WireMockServer for field server
[INFO] [stdout] 2023-02-07 10:45:28,831 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-3) Sent a message to telegram SendLocationMessage{latitude=29.977818', longitude=31.136329', title=Pyramid of Queen Henutsen', address=El-Hussein Ibn Ali Ln, Nazlet El-Semman, Al Haram, Giza Governorate, Egypt', foursquareId=null', foursquareType=null', disableNotification=null', replyToMessageId=null', replyMarkup=null}
[INFO] [stdout] 2023-02-07 10:45:28,845 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (main) Injecting WireMockServer for field server
[INFO] [stdout] 2023-02-07 10:45:28,883 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-3) Sent a message to telegram A message from camel-quarkus-telegram
[INFO] [stdout] 2023-02-07 10:45:28,887 INFO  [org.apa.cam.qua.tes.wir.WireMockTestResourceLifecycleManager] (main) Injecting WireMockServer for field server
[INFO] [stdout] 2023-02-07 10:45:28,923 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-3) Sent a message to telegram SendLocationMessage{latitude=29.974834', longitude=31.138577', livePeriod=120', disableNotification=null', replyToMessageId=null', replyMarkup=null}
[INFO] [stdout] 2023-02-07 10:45:29,505 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-3) Sent a message to telegram {message_id='44', inlineMessageId=null', latitude=29.974928', longitude=31.131115', replyKeyboardMarkup=null}
[INFO] [stdout] 2023-02-07 10:45:30,039 INFO  [org.apa.cam.qua.com.tel.it.TelegramResource] (executor-thread-3) Sent a message to telegram StopMessageLiveLocationMessage{message_id='44', inlineMessageId=null', replyKeyboardMarkup=null}
[ERROR] Tests run: 8, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 101.463 s <<< FAILURE! - in org.apache.camel.quarkus.component.telegram.it.TelegramTest
[ERROR] mp3  Time elapsed: 30.066 s  <<< ERROR!
java.net.SocketTimeoutException: Read timed out
        at java.base/sun.nio.ch.NioSocketImpl.timedRead(NioSocketImpl.java:273)
        at java.base/sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:299)
        at java.base/sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:340)
        at java.base/sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:789)
        at java.base/java.net.Socket$SocketInputStream.read(Socket.java:1025)
        at org.apache.http.impl.io.AbstractSessionInputBuffer.fillBuffer(AbstractSessionInputBuffer.java:161)
        at org.apache.http.impl.io.SocketInputBuffer.fillBuffer(SocketInputBuffer.java:82)
        at org.apache.http.impl.io.AbstractSessionInputBuffer.readLine(AbstractSessionInputBuffer.java:276)
        at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:138)
        at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:56)
        at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:259)
        at org.apache.http.impl.AbstractHttpClientConnection.receiveResponseHeader(AbstractHttpClientConnection.java:294)
        at org.apache.http.impl.conn.DefaultClientConnection.receiveResponseHeader(DefaultClientConnection.java:257)
        at org.apache.http.impl.conn.ManagedClientConnectionImpl.receiveResponseHeader(ManagedClientConnectionImpl.java:207)
        at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:273)
        at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:125)
        at org.apache.http.impl.client.DefaultRequestDirector.tryExecute(DefaultRequestDirector.java:679)
        at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:481)
        at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:835)
        at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
        at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
        at org.apache.http.client.HttpClient$execute$0.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at org.apache.http.client.HttpClient$execute$0.call(Unknown Source)
        at io.restassured.internal.RequestSpecificationImpl$RestAssuredHttpBuilder.doRequest(RequestSpecificationImpl.groovy:2088)
        at io.restassured.internal.http.HTTPBuilder.post(HTTPBuilder.java:350)
        at io.restassured.internal.http.HTTPBuilder$post$3.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:148)
        at io.restassured.internal.RequestSpecificationImpl.sendRequest(RequestSpecificationImpl.groovy:1194)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:578)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
        at groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:166)
        at io.restassured.internal.filter.SendRequestFilter.filter(SendRequestFilter.groovy:30)
        at io.restassured.filter.Filter$filter$0.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at io.restassured.filter.Filter$filter.call(Unknown Source)
        at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
        at io.restassured.filter.time.TimingFilter.filter(TimingFilter.java:56)
        at io.restassured.filter.Filter$filter.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at io.restassured.filter.Filter$filter$0.call(Unknown Source)
        at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
        at io.restassured.filter.FilterContext$next.call(Unknown Source)
        at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1686)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:578)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
        at groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:55)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:203)
        at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1692)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:578)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
        at groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
        at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:55)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:203)
        at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy:180)
        at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy)
        at org.apache.camel.quarkus.component.telegram.it.TelegramTest.mp3(TelegramTest.java:120)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:578)
        at io.quarkus.test.junit.QuarkusTestExtension.runExtensionMethod(QuarkusTestExtension.java:997)
        at io.quarkus.test.junit.QuarkusTestExtension.interceptTestMethod(QuarkusTestExtension.java:816)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
        at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
        at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
[ERROR] mp4  Time elapsed: 30.035 s  <<< ERROR!
...
```

--- a/integration-tests/telegram/src/test/java/org/apache/camel/quarkus/component/telegram/it/TelegramTest.java
+++ b/integration-tests/telegram/src/test/java/org/apache/camel/quarkus/component/telegram/it/TelegramTest.java
@@ -35,7 +35,6 @@ import org.apache.camel.quarkus.test.TrustStoreResource;
 import org.apache.camel.quarkus.test.wiremock.MockServer;
 import org.jboss.logging.Logger;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
 
 import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
 import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
@@ -112,7 +111,6 @@ public class TelegramTest {
     }
 
     @Test
-    @EnabledIfEnvironmentVariable(named = "TELEGRAM_AUTHORIZATION_TOKEN", matches = ".+") //https://github.com/apache/camel-quarkus/issues/4513
     public void mp3() throws IOException {
         try (InputStream in = getClass().getClassLoader().getResourceAsStream("camel-quarkus-rocks.mp3")) {
             /* Send a message */
@@ -126,7 +124,6 @@ public class TelegramTest {
     }
 
     @Test
-    @EnabledIfEnvironmentVariable(named = "TELEGRAM_AUTHORIZATION_TOKEN", matches = ".+") //https://github.com/apache/camel-quarkus/issues/4513
     public void mp4() throws IOException {
         try (InputStream in = getClass().getClassLoader().getResourceAsStream("camel-quarkus-rocks.mp4")) {
             /* Send a message */
@@ -140,7 +137,6 @@ public class TelegramTest {
     }
 
     @Test
-    @EnabledIfEnvironmentVariable(named = "TELEGRAM_AUTHORIZATION_TOKEN", matches = ".+") //https://github.com/apache/camel-quarkus/issues/4513
     public void pdf() throws IOException {
         try (InputStream in = getClass().getClassLoader().getResourceAsStream("camel-quarkus-rocks.pdf")) {
             /* Send a message */

--- a/pom.xml
+++ b/pom.xml
@@ -172,7 +172,7 @@
         <slf4j-log4j12.version>1.7.30</slf4j-log4j12.version><!-- Mess in the transitive dependencies of hbase-testing-util -->
         <sshd.version>2.9.2</sshd.version><!-- @sync io.quarkus:quarkus-bom:${quarkus.version} dep:org.apache.sshd:sshd-common -->
         <stax2.version>4.2</stax2.version>
-        <wiremock.version>3.0.0-beta-2</wiremock.version>
+        <wiremock.version>3.0.0-beta-4</wiremock.version>
         <zt-exec.version>1.12</zt-exec.version>
 
         <!-- Tooling dependency versions (keep sorted alphabetically) -->

